lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/nested-journey-definitions.yaml

```diff
@@ -21,3 +25,5 @@ ADDRESS_AND_FRAUD:
           exitEventToEmit: end
         next:
           exitEventToEmit: next
+        enhanced-verification:
+          exitEventToEmit: next
```

Comment:
This is an interesting one - I think I'd feel slightly more comfortable if this emitted `enhanced-verification` as an exit event...

This will mean mapping `enhanced-verification` from each of the address + fraud steps, but avoids any risk that we ignore the V03 in cases where the address + fraud check happens last.

Reply:
Now https://govukverify.atlassian.net/browse/PYIC-3780